serve/package.go

```diff
@@ -262,6 +264,66 @@ func (*PluginServe) copyDocs(distPath, docsPath string) error {
 	return nil
 }
 
+func (*PluginServe) validatePluginExports(pluginPath string) error {
+	s, err := os.Stat(pluginPath)
+	if err != nil {
+		return err
+	}
+	if !s.IsDir() {
+		return errors.New("plugin path must be a directory")
+	}
+
+	checkRelativeDirs := []string{"resources" + string(filepath.Separator) + "plugin", "plugin"}
+	foundDirs := []string{}
+	for _, dir := range checkRelativeDirs {
+		p := filepath.Join(pluginPath, dir)
+		s, err := os.Stat(p)
+		if err == nil && s.IsDir() {
+			foundDirs = append(foundDirs, dir)
+		}
+	}
+	if len(foundDirs) == 0 {
+		return fmt.Errorf("plugin directory must contain at least one of the following directories: %s", strings.Join(checkRelativeDirs, ", "))
+	}
+
+	versionRegex := regexp.MustCompile(`^\s?Version\s*=`)
```

Comment:
```suggestion
	versionRegex := regexp.MustCompile(`Version\s*=`)
```

otherwise `var Version = "development"` would fail

Reply:
Fixed by adding `(var)?`

` var Version = ` wouldn't work (preceding space before `var` but I think that's fine, or I could add another `\s?`

https://regex101.com/r/7Hm4PT/3

Reply:
will add regex tests.